src/GameWorld.cpp

```diff
@@ -40,21 +42,10 @@ void GameWorld::resetWorld() {
 void GameWorld::updateGameObjects(double deltaTime, double totalTime) {
 	// Keep track of the last spawn time internally to know when to spawn next
 	static double previousSpawnTime = 0.0;
-    static double previousCookieTime = 0.0;
 
-	// Spawn a new bunny every ~3 seconds, and max out at 30 bunnies

```

Comment:
oh shit, I removed that when refactoring my code a little to fast. I'll bring it back